conans/client/remover.py

```diff
@@ -109,23 +108,26 @@ def _local_remove(self, ref, src, build_ids, package_ids):
             self._user_io.out.warn(self._message_removing_editable(ref))
             return
 
-        # Make sure to clean the locks too
-        self._cache.remove_package_locks(ref)
-        remover = DiskRemover(self._cache)
+        # Get the package layout using 'short_paths=False', remover will make use of the
+        #  function 'rm_conandir' which already takes care of the linked folder.
+        package_layout = self._cache.package_layout(ref, short_paths=False)
```

Comment:
I read the comments, but wouldn't be in any case a safer alternative to pass short_paths=None, just in case?

Reply:
By the time of the commit 5e34382 the default value of the `short_paths` argument in thte `package_layout` function was `False` and one test in Windows was failing: https://conan-ci.jfrog.info/blue/organizations/jenkins/ConanTestSuite/detail/PR-4496/9/pipeline

If we use `short_paths=None` the remover will point to the short folders, will empty them, but won't delete the folders themselves.